addons/account/models/account_move.py

```diff
@@ -32,7 +32,6 @@
     is_html_empty,
     create_index,
 )
-from odoo.addons.base_import.models.base_import import FILE_TYPE_DICT
```

Comment:
Can we have excel or spreadsheet here ? If we can maybe we could extend the FILE_TYPE_DICT and using still using instead of creating a new one ?

Reply:
`FILE_TYPE_DICT` is used only in `base_import` and it is totally separate from this functionality, so imho a separate dictionary is ok